roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ingress-nginx-controller-ds.yml.j2

```diff
@@ -27,10 +27,8 @@ spec:
 {% if ingress_nginx_host_network %}
       hostNetwork: true
 {% endif %}
-{% if 'kube-ingress' in groups and groups['kube-ingress']|length > 0 %}
```

Comment:
You made the kube-ingress non optional ?

Reply:
Yes, I made kube-ingress non-optional for deploying ingress-nginx, as commented above